static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -988,7 +988,7 @@ function ReleaseSourceMapMatchingChecklistItem({
           <p>
             {tct(
               'The source file for this stack frame is missing a source map reference. A source map reference is usually represented by a [sourceMappingUrl] comment at the bottom of your source file.',
-              {sourceMappingUrl: <MonoBlock>//# sourceMappingURL=...</MonoBlock>}
```

Comment:
The linter thought this was a comment and suggested wrapping it with `{/* .. */}`. But it's a string example we want to show so i worked around the linter.